Dockerfile

```diff
@@ -83,11 +86,11 @@ ENV HYRAX_ENGINE_PATH /app/samvera/hyrax-engine
 COPY --chown=1001:101 $APP_PATH /app/samvera/hyrax-webapp
 COPY --chown=1001:101 . /app/samvera/hyrax-engine
 
-RUN cd /app/samvera/hyrax-engine && bundle install --jobs "$(nproc)"
```

Comment:
sorry that it's taken me so long to come around to this. in my testing, this leaves the Hyrax dev environment broken/unready.

after pulling this and building `docker-compose exec -w /app/samvera/hyrax-engine app sh -c "rubocop"` gives `sh: rubocop: not found`, and similar results for `rspec` and other common development commands.

i think we really do want to bundle for the engine, in this context.

Reply:
Okay, I've edited the PR description and will work on this more.